src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -209,10 +209,14 @@ const char* const kReservedWordList[] = {
 hash_set<string> kReservedWords =
     MakeWordsMap(kReservedWordList, GOOGLE_ARRAYSIZE(kReservedWordList));
 
-string SanitizeNameForObjC(const string& input, const string& extension) {
+string SanitizeNameForObjC(const string& input,
```

Comment:
wouldn't it be better to return the boolean and modify the string with the pointer?